### PR TITLE
[sobjectizer] update to v5.7.2.3

### PR DESCRIPTION
**Pull request by eao197**
SObjectizer updated to the latest version.

### CODE REVIEW

File: ports/sobjectizer/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "sobjectizer",
-  "version-string": "5.7.2.2",
+  "version-string": "5.7.2.3",
```
Comment (Cheney-W) [inline ports/sobjectizer/vcpkg.json:3]:
```suggestion
  "version-semver": "5.7.2.3",
```

**Review by Cheney-W — CHANGES_REQUESTED** [review]

File: ports/sobjectizer/vcpkg.json (hunk 1 above)
Comment (eao197) [reply ports/sobjectizer/vcpkg.json:3]: I've got the following error:
```
$./vcpkg install sobjectizer
Computing installation plan...
Errors occurred while parsing /home/eao197/sandboxes/vcpkg.eao197/ports/sobjectizer/vcpkg.json
    $ (vcpkg.json): 'version-semver' text was not a semantic version:
Error: String `5.7.2.3` is not a valid Semantic Version string, consult https://semver.org
Error: failed to load port from /home/eao197/sandboxes/vcpkg.eao197/ports/sobjectizer
```

Comment (Cheney-W) [reply ports/sobjectizer/vcpkg.json:3]: Sorry, this is my mistake. According to the docs: https://github.com/microsoft/vcpkg/blob/master/docs/specifications/versioning.md#22-package-versions, I think this item should be "version": "5.7.2.3".

Comment (eao197) [reply ports/sobjectizer/vcpkg.json]: I've changed `version-string` to `version`.

Comment (Cheney-W) [reply ports/sobjectizer/vcpkg.json]: Thank you!
BTW, please run the command: vcpkg x-add-version sobjectizer --overwrite-version to update the json file again.

Comment (eao197) [reply ports/sobjectizer/vcpkg.json]: Done.

Comment (Cheney-W) [reply ports/sobjectizer/vcpkg.json]: Thank you!

**Review by Cheney-W — APPROVED** [review]

**Comment by ras0219-msft** [comment]
LGTM, thanks for the PR!

**Pull request merged**